src/KuduClient.ts

```diff
@@ -25,6 +25,25 @@ export class KuduClient {
         });
     }
 
+    getFile(path: string): Promise<string> {
```

Comment:
FYI - I'm okay with you addressing this comment in a separate PR. It might not be a trivial change and I don't want to delay this for a long time

Reply:
It seems like it might take a bit of refactoring to accomplish this.  I made an issue for this and will take care of it after release.

https://github.com/Microsoft/vscode-azureappservice/issues/237